apis/python/test/test_ingestion.py

```diff
@@ -154,22 +176,25 @@ def test_ivf_flat_ingestion_fvec(tmp_path):
         source_type=source_type,
         partitions=partitions,
     )
-    result = np.transpose(
-        index.query(np.transpose(query_vectors), k=k, nprobe=partitions)
-    )
+    result = np.transpose(index.query(np.transpose(query_vectors), k=k, nprobe=nprobe))
     assert accuracy(result, gt_i) > MINIMUM_ACCURACY
 
     index_ram = IVFFlatIndex(uri=array_uri, dtype=dtype)
     result = np.transpose(
-        index_ram.query(np.transpose(query_vectors), k=k, nprobe=partitions)
+        index_ram.query(np.transpose(query_vectors), k=k, nprobe=nprobe)
     )
     assert accuracy(result, gt_i) > MINIMUM_ACCURACY
     result = np.transpose(
         index_ram.query(
             np.transpose(query_vectors),
             k=k,
-            nprobe=partitions,
+            nprobe=nprobe,
             use_nuv_implementation=True,
         )
     )
     assert accuracy(result, gt_i) > MINIMUM_ACCURACY
+
+    result = index_ram.distributed_query(
+        np.transpose(query_vectors), k=k, nprobe=nprobe, mode=Mode.LOCAL
```

Comment:
Eventually we should harmonize the interfaces so that we don't need to transpose everywhere.

Reply:
Done